tests/test/builtin_impls.rs

```diff
@@ -0,0 +1,604 @@
+use super::*;
```

Comment:
It might be worth splitting this up as a module. So `copy`, `clone`, `unsize`, etc.